.github/workflows/test.yml

```
@@ -19,38 +19,63 @@ jobs:
       id: action-ros-ci
       uses: ros-tooling/action-ros-ci@v0.2
```

by the way, you could bump this to `v0.3` if you want.

```suggestion
      uses: ros-tooling/action-ros-ci@v0.3
```